.gitlab-ci.yml

```diff
@@ -13,7 +13,7 @@ stages:
 
 variables:
   GIT_STRATEGY:                    fetch
-  GIT_DEPTH:                       3
+  GIT_DEPTH:                       100
```

Comment:
Why is this increased to such a huge number? Also it seems unrelated to the PR.

Reply:
also checking one theory, will place WIP here to solve some issues